### DISCUSSION TITLE
Want to send email notification to the added participants in the Invite-only batch or in the discussion forum.

### DISCUSSION THREAD

**Discussion by snehalsabade** [Sunbird-Ed/Community#620] [General Discussion]
![Notification bell icon](https://github.com/Sunbird-Ed/Community/assets/132333617/5961d035-80a2-4373-aaa0-f8cf5cadf0eb)
Hello all,
**Sunbird Version - 4.10.0**

Please let me know, Is there any document is available or API (curl) is available for send an email notification to the added participants in the Invite-only batch or in the discussion forum?
2. When we added to the particular participants in the batch that participants got invitation by the mail but the participant did not get notification on bell icon?
Please help me in this

CC:@komalm @Renuka-S @vinod303 @paritshivani

**Comment by ashokreddy1208** [comment]
@AmiableAnil @rajeevsathish pls look into this.

**Reply by AmiableAnil** [reply]
@snehalsabade 

1. This is the API to send the [email notification](https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service/apis/notification-apis)
2. After adding the participant only sending the email notification only. Not triggering user feed.

**Comment by ashokreddy1208** [comment]
@snehalsabade  let us know if there is any additional help needed from the community.

**Comment by snehalsabade** [comment]
Is there any API which will trigger user feed, so user got notification on bell icon?

CC @ashokreddy1208 @komalm @Renuka-S @vinod303 @paritshivani

**Reply by AmiableAnil** [reply]
@snehalsabade Please find the API to create the feed here https://lern.sunbird.org/learn/product-and-developer-guide/notification-service/apis#notification-v2-send

**Comment by ashokreddy1208** [comment]
@snehalsabade  let us know if there is any additional help needed from the community. If the issue is resolved then can you mark this thread as answered

**Reply by snehalsabade** [reply]
Thank you for your help.
